HW/10.27/Web_10.27DZ/Web_10.27DZ/Controllers/HomeController.cs

```diff
@@ -25,47 +25,67 @@ public HomeController(IConfiguration configuration, IJsonIteractor jsonIteractor
             _peopleStorage = peopleStorage;
         }
 
-        [HttpPost("PutPeople")]
-        public IActionResult PutPeople()
-        {
-            _jsonIteractor.JsonWriteList(_configuration, Constants.people);
-            return new EmptyResult();
-        }
-
-        [HttpGet("GetPeople")]
+        [HttpGet("all-people")]
         public IActionResult GetPeople()
         {
-            _peopleStorage.innerCol = _jsonIteractor.JsonReadList(_configuration);
+            _peopleStorage.InnerCol = _jsonIteractor.JsonReadList(_configuration);
```

Comment:
I don't really get this
on HttpGet we should only return data and dont save anything or change any state

Reply:
It is better to do this in ctor of PeopleStorage